packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -50,107 +51,120 @@ Default.args = {
+            kind="critical"
+            layout="floating"
+        />
+    </View>
+);
+
+Kinds.parameters = {
+    docs: {
+        storyDescription: `Banners have four possible kinds (\`kind\` prop) -
+            info (default), success, warning, and critical. Info is blue
+            with an info "i" icon, success is green with a smiling icon,
+            warning is yellow with a triangular "!" icon, and critical is
+            red with a round "!" icon.`,
+    },
+};
+
+export const Layouts: StoryComponentType = () => {
+    const borderStyle = {border: `2px solid ${Color.pink}`};
+    const floatingContainerStyle = {padding: Spacing.xSmall_8};
+
+    return (
+        <View style={styles.container}>
+            <Banner
+                text="This banner has full-width layout."
+                layout="full-width"
+                kind="success"
+            />
+            <Strut size={Spacing.medium_16} />
+            <Banner
+                text="This banner has floating layout."
+                layout="floating"
+                kind="success"
+            />
+            <Strut size={Spacing.medium_16} />
+            <View style={borderStyle}>
                 <Banner
-                    text="kind: info - This is a message about something informative like an announcement."
-                    kind="info"
+                    text="This banner has full-width layout. There is no space around it."
                     layout="full-width"
-                />
-            </View>
-            <View style={styles.spacing}>
-                <Banner
-                    text="kind: success - This is a message about something positive or successful!"
                     kind="success"
-                    layout="full-width"
                 />
             </View>
-            <View style={styles.spacing}>
+            <Strut size={Spacing.medium_16} />
+            <View style={[borderStyle, floatingContainerStyle]}>
                 <Banner
-                    text="kind: warning - This is a message warning the user about a potential issue."
-                    kind="warning"
-                    layout="full-width"
-                />
-            </View>
-            <View style={styles.spacing}>
-                <Banner
-                    text="kind: critical - This is a message about something critical or an error."
-                    kind="critical"
-                    layout="full-width"
-                />
-            </View>
-        </View>
-        <View style={styles.variantColumn}>
-            <LabelMedium>Floating layout</LabelMedium>
-            <View style={styles.spacing}>
-                <Banner
-                    text="kind: info - This is a message about something informative like an announcement."
-                    kind="info"
+                    text="This banner has floating layout. Space has been added around it."
```

Comment:
I would add to this sentence to say that space has been added around it by an outside component that isn't part of the Banner itself. And that we always want the space added around it, as indicated by a notation in the designs, but that it's up to the developer to ensure that this rule is adhered to on the screen... or something like that... maybe not in the Banner itself, but in a note below it? I do see it somewhat explained in the docs, but are those on a tab that someone might not see unless clicked? It's also not _super_ obvious that it's a separate view adding the padding. Maybe make that view have a dashed-line border to really call it out?

Reply:
OMG!! I read the code and added the comment before clicking into the Storybook!!!! IGNORE ME!!! YOU DID THIS! 🤦‍♀️

Reply:
Yeah, the "docs text" is right there above the story, but I would _maaaaybe_ add a tiny bit to that text blob that does explicitly call out that the "spacing added" is a separate view managed by the developer? Because, to me, that's the critical difference. Not just that there should be a space, but that it has to manually added. 😄